src/readcalls.c

```diff
@@ -191,8 +136,11 @@ int readcalls(const char *logfile) {
 	// remember logline in qsos[] field
 	g_strlcpy(qsos[linenr], inputbuffer, sizeof(qsos[0]));
 	linenr++;
+	// FIXME check for overflow....
```

Comment:
Above two lines can be dropped as soon as we use the qso_array to keep all information.

Reply:
True.